tests/Integration/Service/AuthorizationServiceTest.php

```diff
@@ -0,0 +1,106 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Answear\BoxNowBundle\Tests\Integration\Service;
+
+use Answear\BoxNowBundle\Exception\RequestException;
+use Answear\BoxNowBundle\Response\AuthorizationResponse;
+use Answear\BoxNowBundle\Service\AuthorizationService;
+use Answear\BoxNowBundle\Tests\Util\FileTestUtil;
+use GuzzleHttp\Psr7\Response;
+use Symfony\Component\HttpFoundation\Response as SymfonyResponse;
+
+class AuthorizationServiceTest extends ServiceTestCase
+{
+    public function testSuccessfulAuthorization(): void
```

Comment:
In class above (ConfigurationTest) you used annotations to mark method as tests, here you used method name prefix. Choose one way please ;)